pyscaffold/templates/skeleton.template

```diff
@@ -6,9 +6,9 @@ console script. To run this script uncomment the following line in the
 entry_points section in setup.cfg:
 
     console_scripts =
-        hello_world = ${package}.module:function
-
-Then run `python setup.py install` which will install the command `hello_world`
+     fibonacci = my_project.skeleton:run
```

Comment:
I think we still need ``${package}`` instead of ``my_project`` for this to work.